core_backend/app/admin/routers.py

```diff
@@ -32,6 +33,7 @@ async def healthcheck(
         url = urlparse(ALIGN_SCORE_API)
         healthcheck_url = f"{url.scheme!r}://{url.netloc!r}/healthcheck"
         http_client = get_http_client()
+        assert isinstance(http_client, aiohttp.ClientSession)
```

Comment:
Is this needed if we ensure get_http_client always returns ClientSession?

Reply:
i believe so---i'm not sure how the original `def get_http_client()` passed `mypy` checks but without the assertions it is failing pre-commit for me.